sssd_test_framework/utils/gdm.py

```diff
@@ -0,0 +1,80 @@
+"""Manage GDM interface from SCAutolib."""
+
+from __future__ import annotations
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.cli import CLIBuilder, CLIBuilderArgs
+from pytest_mh.conn import ProcessResult
+from pytest_mh.utils.fs import LinuxFileSystem
+
+__all__ = [
+    "GDM",
+]
+
+
+class GDM(MultihostUtility[MultihostHost]):
+    """
+    Call commands from scautolib.
+    """
+
+    def init(self) -> None:
+        """
+        Initialize GDM for testing
+        """
+        self.host.conn.exec(["/opt/test_venv/bin/scauto", "gui", "init"])
```

Comment:
IMO, this should be handled when any other method is called and keep in memory whether `scauto` has been initialized.

Reply:
Still applies

Reply:
I implemented this.  Take a look and let me know if I missed something.